### ISSUE TITLE
"You cannot perform that action on the specified target" when attempting to use spells or job abilities on self

### ISSUE THREAD

**Issue by VirtuosusFFXI** [EdenServer/community#2158]
### Checklist

- [ ✓] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [✓ ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ✓] I have provided my client version in the details. (type /ver into your game window)

### Details

Not entirely sure what the cause of this occuring is, but it has caused me multiple deaths already. It's happening during my mining ventures in both Halvung and Mount Zhayolm. There are situations where I might get aggro while mining a node because I wasn't paying attention, and I'll get hit a bit and sleep the mob and go to cure myself, and then get the message in the title, interrupting my spell. Attempting to use any job abilities results in nothing happening. The JA does not go on cooldown and it will not go off no matter what. The only solution I've found is death. Upon returning to town, I can use spells on myself again and job abilities go through without issue. 

![Lin_2019 10 25_023044](https://user-images.githubusercontent.com/49970272/67549050-579cc380-f6d1-11e9-8ffc-0368865a6e75.png)

**Comment by Setz0r** [comment]
were you right in the middle of a mining animation when getting hit?

**Comment by godmode** [comment]
Thanks for reporting. This has to do with the event handler not finishing. A character is locked from being aggroed or being buffed when in an event. Leaving this open because it is a problem but the only solution is to zone or log in and out currently.

**Comment by VirtuosusFFXI** [comment]
Yeah Setz, I was. I figured it had something to do with the mining animation being interrupted, thanks for the reply.

**Comment by VirtuosusFFXI** [comment]
Happened again, this time in Promyvion - Vahzl. No event was interrupted this time, nothing particularly special happened in the 5 minutes prior. I was able to cast on myself and suddenly I wasn't anymore. Same message, same character.

**Comment by marlinla** [comment]
Happened to me while mining

**Comment by ThoseWhoFight** [comment]
Can confirm this is still happening. It seems to be when a person is in the action of mining a node, then either perform an action again i.e. clicking a macro to target the node and confirming while mid mining already, or having an action performed on said mining character that causes this bug. Afterward you can cast offensive spells on other targets, but you cannot continue mining or cast self targeting spells without zoning or relogging. Unfortunately this bug also causes you to be unable to logout in some circumstances, so your only option is to force close client and pray you're still alive when you log back in.
